src/main/java/me/botsko/prism/utils/BlockUtils.java

```diff
@@ -122,4 +122,10 @@ public static int blockIdMustRecordAs(int block_id) {
         final Material[] materials = { Material.WATER, Material.STATIONARY_WATER };
         return removeMaterialsFromRadius( materials, loc, radius );
     }
+
+    public static boolean isDoor(Material m) {
```

Comment:
This method is not being used anywhere in this change set, as far as I can find...